save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/domain/Sdk.kt

```diff
@@ -52,8 +55,8 @@ fun String.toSdk(): Sdk {
     require(splitSdk.size == 2) { "Cant find correct sdk and version" }
     val (sdkType, sdkVersion) = splitSdk.run { this.first() to this.last() }
     return when (sdkType) {
-        Jdk.NAME -> Jdk(sdkVersion)
-        Python.NAME -> Python(sdkVersion)
+        Jdk.NAME, Jdk("-1").name -> Jdk(sdkVersion)
```

Comment:
there is `Jdk.NAME` and `Python.NAME` -- can we use them? I mean that there are two values which mean similar option

Reply:
Yeah, there are string values for two purposes: display name, which is currently `Companion.NAME`, and docker image name, which is in the property `name`. This is indeed confusing, but let's create a separate issue to refactor it, because this PR is already pretty large